assets/src/components/settingsPage.tsx

```diff
@@ -27,7 +27,7 @@ const SettingsPage = (): ReactElement<HTMLDivElement> => {
   const mobileMenuClass = mobileMenuIsOpen ? "blurred-mobile" : ""
 
   return (
-    <div className={`c-page c-page--settings ${mobileMenuClass}`}>
+    <div className={`l-page m-settings-page ${mobileMenuClass}`}>
```

Comment:
**question:** it looks like (if I'm using `git grep` correctly) there is no actual styling for the `c-page--settings` / `m-settings-page` classes. Does it make sense to keep this class if there's no actual styling associated with it? I don't have a strong opinion but I'd lean slightly in the direction of avoiding classnames that aren't used somewhere.

Reply:
I'm glad you caught that! Happy to remove.

Reply:
Actually changing my answer: I think it is still worth including since there are descendent like `m-settings-page__section-header` exist, so I think we should have a class name to define the Block. I'll rename to `c-settings-page` though